### DISCUSSION TITLE
Login as me / self

### DISCUSSION THREAD

**Discussion by ultrono** [spatie/laravel-login-link#30] [Ideas]
Hi,

Pretty much every time I've used this package I end up creating a login link that renders "Login as me", where I specify my own email address.

Would you be open to a PR that creates an extra component that creates an extra Blade attribute (i.e. `<x-login-link self="true"/>`) that would render the email I set in in a configuration value?

Appreciate if you're the sole developer on a project you can easily create a link like this. However, I'm working on a project at the moment where there are multiple developers and having an easy single configuration change to create a self login link would be great.

For the moment, I always create the latter within my project without touching the package, but having such functionality as part of the package may be good.

So at present I add a custom value to the published package configuration file and I always end up with:

```blade
 <x-login-link :email="config('login-link.my-email-address')" label="Login as me" />
```

So, with my proposed PR you'd simply do (which would fetch a standard configuration item from this package):

```blade
<x-login-link me="true" />
```

Would you be open to a PR on this?

Thanks

**Comment by freekmurze** [comment]
If the PR is simple, tested and well documented, I'll consider merging it in.
